.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
Better to stick with what we have no main repo

Reply:
you mean what we have in our main repo?  I'll pop that on over, if that's what you mean

Reply:
Yes, from main repo.

Reply:
excellent.  I'll do just that